ObliviousHarmony/vscode-php-codesniffer
# Improved Executable Argument Parsing

### All Submissions:


### Changes proposed in this Pull Request:

This pull request adds the `split-string` package to replace the regex parsing of executable paths. This will provide more comprehensive quoting behavior across operating systems and better cover Windows paths.

Closes https://github.com/ObliviousHarmony/vscode-php-codesniffer/issues/97.

### How to test the changes in this Pull Request:

1. Add arguments to an executable option.
2. Confirm that they work correctly.
